iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -77,12 +79,14 @@ UnixDomainSocket::UnixDomainSocket(const IpcChannelName_t& name,
 {
 }
 
-// NOLINTNEXTLINE(readability-function-size) todo(iox-#832): make a struct out of arguments
+// @todo(iox-#832): make a struct out of arguments
+// NOLINTBEGIN(readability-function-size, bugprone-easily-swappable-parameters)
 UnixDomainSocket::UnixDomainSocket(const NoPathPrefix_t,
```

Comment:
I wonder how this can be merged with the other ctor, the only difference is the tag type.